app/views/hyrax/admin/collection_types/_form_participants.html.erb

```diff
@@ -1,7 +1,7 @@
 <%# OVERRIDE Hyrax v3.4.2 Make group field a select instead of text input %>
 <h3><%= t('.add_participants') %></h3>
 <p><%= t('.instructions') %></p>
-<% access_options = options_for_select([['Manager', Hyrax::CollectionTypeParticipant::MANAGE_ACCESS], ['Creator', Hyrax::CollectionTypeParticipant::CREATE_ACCESS]]) %>
+<% access_options = options_for_select([[t('hyrax.dashboard.collections.form_share.access_options.manager'), Hyrax::CollectionTypeParticipant::MANAGE_ACCESS], [t('hyrax.dashboard.collections.form_share.access_options.creator'), Hyrax::CollectionTypeParticipant::CREATE_ACCESS]]) %>
```

Comment:
Can we break this one long line into multiple shorter ones to make it easier to read?